.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -188,7 +188,6 @@ jobs:
           #   occurring in the meanwhile.
           CILIUM_INSTALL_DEFAULTS=" \
             --set=debug.enabled=true \
-            --set=bpf.monitorAggregation=medium \
```

Comment:
Nit. It would make sense to drop the monitor aggregation related comment above, as no longer relevant. Not sure it is worth going through CI once more though.

Reply:
Fighting with CI to make CI better :smiley: 
I would prefer to merge it asap to have this improvement for everyone, comment can be fixed later on.